docs/how-to/get-package-source.rst

```diff
@@ -1,194 +1,241 @@
+  .. code-block:: bash
 
-    $ pull-ppa-source --ppa dviererbe/hello hello 2.10-3
+      pull-ppa-source --ppa dviererbe/hello hello 2.10-3
 
-``pull-debian-source``
-^^^^^^^^^^^^^^^^^^^^^^
+:command:`pull-debian-source`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Examples
-""""""""
+* To download the latest version of the ``hello`` source package from
+  :term:`Debian`:
 
-download the latest version of the ``hello`` package from debian::
+  .. code-block:: bash
 
-    $ pull-debian-source hello
+      pull-debian-source hello
 
-download the latest version of the ``hello`` package for the ``sid`` release from debian::
+* To download the latest version of the ``hello`` source package for the
+  ``sid`` release from Debian:
 
-    $ pull-debian-source hello sid
+  .. code-block:: bash
 
-download the version ``2.10-3`` of the ``hello`` package from debian::
+      pull-debian-source hello sid
 
-    $ pull-debian-source hello 2.10-3
+* To download the version ``2.10-3`` of the ``hello`` source package from Debian:
 
-``apt-get source``
-------------------
+  .. code-block:: bash
 
-The APT package manager can also fetch source packages via ``apt-get source``.
+      pull-debian-source hello 2.10-3
+
+:command:`apt-get source`
+-------------------------
+
+The :term:`APT` :term:`Package Manager` can also fetch source packages.
 
 .. important::
 
-   Source packages are tracked separately from binary packages via ``deb-src``
-   lines in the ``sources.list(5)`` file. This means that you will need to add
-   such a line for each repository you want to get sources from; otherwise
-   you will probably get either the wrong (too old/too new) source versions
-   or none at all.
+   Source packages are tracked separately from
+   :term:`binary packages <Binary Package>` via ``deb-src`` lines in the
+   :manpage:`sources.list(5)` file. This means that you will need to add
+   such a line for each :term:`repository<Repository>` you want to get sources
+   from; otherwise you will probably get either the wrong (too old/too new)
+   source versions -- or none at all.
 
-Basic usage::
+Basic usage
+^^^^^^^^^^^
 
-    $ apt-get source <PACKAGE>
+.. code-block:: bash
 
-Example::
+    apt-get source <PACKAGE>
+
+Example
+^^^^^^^
+.. code-block:: bash
 
-    $ apt-get source hello
+    apt-get source hello
 
-You can find further information on the `man page <AptGetManPage_>`_.
+You can find further information on the manual page :manpage:`apt-get(8)`.
 
 ``dget``
 --------
 
-The ``dget`` command is part of the ``devscripts`` package. If you call it with 
-the URL of a ``.dsc`` or ``.changes`` file it acts as a source package aware ``wget``.
+The :command:`dget` command is part of the ``devscripts`` package. If you call
+it with the URL of a ``.dsc`` or ``.changes`` file it acts as a source package
+aware :manpage:`wget(1)`.
```

Comment:
I found this wording "source package aware wget" to be very confusing and I'm not sure what you meant to say here.

Reply:
Agree, I expanded the explanation.

Resolved with fd05113bb2ab1fc82184e4ef5dba4ce4b3afaddc